src/_includes/layouts/single--resource.njk

```diff
@@ -29,15 +29,15 @@
                     {% for topic in topics %}
                         <div class="card-tag">
                             <svg role="presentation"><use xlink:href="#topic" /></svg>
-                            <p>{{ topic | getResourceMetadataLabel('topics') | safe }}</p>
```

Comment:
This is added as a filter in preview.js. So I'm not sure if the solution should be to remove this or to add to eleventy config. That being said there seems to be several filters in preview.js that aren't used. Do you know if that file was copied over from somewhere else?

Reply:
I think some filters added to the preview is for potential use cases, so not all of them are being used. I tried adding getResourceMetadataLabel in eleventy config, but it was failing because inside of getResourceMetadataLabel is using import keywords to get required json files. I also think it is not possible to have topic data being empty, so I thought it is fine to remove the use of the filter.

Reply:
@chosww there's something wrong with the resources page in general for which these layout appears to be used. For example if you go to the page for [Infusion](https://idrc.ocadu.ca/resources/infusion/), you can see it has Infusion in the header but otherwise still looks like the Resources landing page. So in short I guess this change is okay for now, but really there probably needs to be a bigger fix or more cleanup. Can you file a ticket to look into the single resource page. They should either not be published or should display the correct contents.

Reply:
https://github.com/inclusive-design/idrc/issues/921